runtime/gc_modron_standard/ContinuationObjectBufferStandard.cpp

```diff
@@ -81,3 +81,18 @@ MM_ContinuationObjectBufferStandard::flushImpl(MM_EnvironmentBase* env)
 		_continuationObjectListIndex = 0;
 	}
 }
+
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+bool
+MM_ContinuationObjectBufferStandard::reinitializeForRestore(MM_EnvironmentBase *env)
+{
+	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(env);
+
+	Assert_MM_true(_maxObjectCount > 0);
+	Assert_MM_true(extensions->objectListFragmentCount > 0);
+
+	_maxObjectCount = extensions->objectListFragmentCount;
```

Comment:
this line is basically re-constructing the object, so you could just call a constructor, but up to if you like it

new(this) MM_ContinuationObjectBufferStandard(extensions, extensions->objectListFragmentCount);

Reply:
Prefer to leave this to a simple assignment, we're essentially just updating the `_maxObjectCount ` int

Reply:
I'm fine in this simple case with leaving as-is, but in more general/complex scenario re-init code should try to share original init code (including constructor).